themes/classic/templates/_partials/form-fields.tpl

```diff
@@ -99,9 +119,15 @@
           name="{$field.name}"
           type="{$field.type}"
           value="{$field.value}"
+          placeholder="{if isset($field.availableValues.placeholder)}{$field.availableValues.placeholder}{/if}"
```

Comment:
`placeholder` attribute name can be removed as well if no value is provided by the controller.